graylog2-web-interface/src/components/common/EntityDataTable/EntityDataTable.tsx

```diff
@@ -127,6 +134,10 @@ type Props<Entity extends EntityBase> = {
   onColumnsChange: (columnIds: Array<string>) => void,
   /** Function to handle sort changes */
   onSortChange: (newSort: Sort) => void,
+  /** Function to handle page size changes */
+  onPageSizeChange?: (newPageSize: number) => void,
+  /** Active page size */
+  pageSize?: number
```

Comment:
Instead of implementing the page size select in the entity data table I thought about providing the whole select component as a prop, but we would have to do this every time we use the entity data table and we would have to repeat ourselfs.

Reply:
I would assume that pagination is a central responsibility for the entity data table, so presenting (optional!) props for this is totally fine.